tests/python/contrib/test_ethosu/test_codegen.py

```diff
@@ -242,6 +242,93 @@ def representative_dataset():
     infra.verify_source(compiled_models, accel_type)
 
 
+@pytest.mark.parametrize("weight_min, weight_max", [(0.0, 1e-11), (-1e10, 1e10)])
+def test_out_of_range_scaling(weight_min, weight_max):
+    ifm_shape = (1, 6, 6, 2)
+    dtype = "int8"
+    strides = (1, 1)
+    kernel_shape = (1, 1)
+    dilation = (1, 1)
+    padding = "SAME"
+    activation = "RELU"
+    accel_type = "ethos-u55-128"
+
+    def create_tflite_graph():
+        class Model(tf.Module):
+            @tf.function
+            def tf_function(self, x):
+                # Use tf.nn API to create the model
+                tf_strides = [1, strides[0], strides[1], 1]
+                weights = np.random.uniform(size=[kernel_shape[-1], kernel_shape[1], 2, 2])
```

Comment:
It doesn't make an effective difference here, but I think `kernel_shape[-1]` and `kernel_shape[1]` refer to the same element

Reply:
You are right, I fixed this and refactored to use Matthew's functions